pkgs/by-name/tu/turn-rs/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+}:
+
+let
+  pname = "turn-rs";
+  version = "3.0.0";
+  src = fetchFromGitHub {
+    owner = "mycrl";
+    repo = "turn-rs";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-AtNvvL/zNqqfCMVG2o8zrT+a1O6BkvKFf1nsvUDrvWs=";
+  };
+in
+rustPlatform.buildRustPackage {
+  inherit pname version src;
```

Comment:
This can use finalAttrs IIRC

Reply:
> It only work on mkDeviration.

That's not true. Some other helper functions have support for it but maybe rust's not yet.